tests/unit/test_main.py

```diff
@@ -246,5 +248,69 @@ def test_preference_args(self):
         prefs = main.preference(None, prefs_args)
         self.assertEquals(len(prefs), 0)
 
+
+class TestCheckMozregresionVersion(unittest.TestCase):
+    @patch('requests.get')
+    @patch('sys.stdout')
+    def test_pypi_is_accesible(self, stdout, get):
+        options = None
+        logger = Mock('mozlog.structured.commandline.setup_logging')(
+                      "mozregression",
+                      options,
+                      {"mach": stdout})
+        response = Mock(json=lambda: {'info': {'version':  __version__}})
+        get.return_value = response
+        main.check_mozregression_version(logger)
+        self.assertFalse(logger.critical.called)
+
+    @patch('requests.get')
+    @patch('sys.stdout')
+    def test_RequestException_error(self, stdout, get):
+        options = None
+        logger = Mock('mozlog.structured.commandline.setup_logging')(
+                      "mozregression",
+                      options,
+                      {"mach": stdout})
+        get.side_effect = requests.RequestException
+        main.check_mozregression_version(logger)
+        self.assertRaises(requests.RequestException, get)
+
+    @patch('requests.get')
+    @patch('sys.stdout')
+    def test_key_error(self, stdout, get):
```

Comment:
Well, this seems not really useful to me. the code executed is the same for the previous test. If you want to test every kind of handled exception, maybe write a loop in one test method. or jut get rid of this test method :)